ProcessLib/Process.h

```diff
@@ -147,13 +141,22 @@ class Process
         return _extrapolator_data.getDOFTable();
     }
 
+    /// Initialize the boundary conditions for single PDE. It is called by
+    /// initializeBoundaryConditions().
+    void initializeBoundaryConditionPerPDE(
+        const NumLib::LocalToGlobalIndexMap& dof_table, const int process_id);
```

Comment:
can be private, I guess...

Reply:
It is also called by  HydroMechanicsProcess, the derived class. Therefore, it is in the protected section.